common/client-libs/validator-client/src/validator_api/mod.rs

```diff
@@ -2,13 +2,15 @@ pub mod error;
 
 use crate::validator_api::error::ValidatorAPIClientError;
 
-use serde::Deserialize;
+use serde::{Deserialize, Serialize};
 use url::Url;
 
 pub const VALIDATOR_API_PORT: u16 = 8080;
 pub const VALIDATOR_API_CACHE_VERSION: &str = "/v1";
 pub(crate) const VALIDATOR_API_MIXNODES: &str = "/mixnodes";
 pub(crate) const VALIDATOR_API_GATEWAYS: &str = "/gateways";
+pub const VALIDATOR_API_VERIFICATION_KEY: &str = "/verification_key";
+pub const VALIDATOR_API_BLIND_SIGN: &str = "/blind_sign";
```

Comment:
This comment is not specifically for this, but perhaps we should get rid of all slashes in constants and instead only introduce them in urls directly? That way we'd prevent future double slash issues

Reply:
I'm not sure, but I think that's how `rocket` mounts them, which is why I left them with the slash prefix, so that I wouldn't need for `format!` the mount arguments